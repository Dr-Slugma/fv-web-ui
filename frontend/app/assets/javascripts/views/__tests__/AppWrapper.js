// A11Y
// const { axe, toHaveNoViolations } = require('jest-axe')
// expect.extend(toHaveNoViolations)
// import ReactDOMServer from 'react-dom/server'

// Standard
import React from 'react'
import ReactDOM from 'react-dom'

// Component to test
// import { JestTestSetup } from 'views/components/JestTestSetup'

// import injectTapEventPlugin from 'react-tap-event-plugin'
import ThemeManager from 'material-ui/lib/styles/theme-manager'
import FirstVoicesTheme from 'views/themes/FirstVoicesTheme.js'
import ConfGlobal from 'conf/local.js'

// Providers provide reducers and actions
import providers from '../../providers/index'

// Views
import AppWrapper from '../AppWrapper'

// require('!style-loader!css-loader!normalize.css')
// require('!style-loader!css-loader!alloyeditor/dist/alloy-editor/assets/alloy-editor-ocean-min.css')
// require('!style-loader!css-loader!tether-shepherd/dist/css/shepherd-theme-arrows.css')
// require('bootstrap/less/bootstrap')
// require('styles/main')

// injectTapEventPlugin()

const context = {
  providers,
  /*combinedProviders: [
      providers // OK for all providers to share the same store for now, make sure actions are unique
    ],*/
  providedState: {
    properties: {
      title: ConfGlobal.title,
      pageTitleParams: null,
      domain: ConfGlobal.domain,
      theme: {
        palette: ThemeManager.getMuiTheme(FirstVoicesTheme),
        id: 'default',
      },
    },
  },
}
describe('AppWrapper', () => {
  test('Mounts', () => {
    // Structure: Arrange
    const container = document.createElement('div')
    ReactDOM.render(<AppWrapper {...context} />, container)

    expect(container.querySelector('#pageNavigation').textContent).toMatch('SIGN IN')
    expect(container.querySelector('#pageFooter').textContent).toMatch('support@fpcc.ca')

    // Structure: Act
    // Structure: Assert
  })

  // test('Accessibility', async() => {
  //   const html = ReactDOMServer.renderToString(<AppWrapper {...context} />)
  //   const results = await axe(html)
  //   expect(results).toHaveNoViolations()
  // })
})
