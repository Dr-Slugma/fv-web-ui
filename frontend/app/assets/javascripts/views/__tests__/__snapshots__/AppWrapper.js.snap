// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppWrapper Snapshot 1`] = `
<div
  data-reactid=".1"
  id="AppWrapper"
>
  <div
    data-reactid=".1.0"
  >
    <div
      class="row"
      data-reactid=".1.0.1"
      id="pageNavigation"
    >
      <div
        data-reactid=".1.0.1.0"
      >
        <div
          data-reactid=".1.0.1.0.0"
          style="background-color:#b40000;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;box-sizing:border-box;font-family:Arial, sans-serif;-webkit-tap-highlight-color:rgba(0,0,0,0);box-shadow:0 1px 6px rgba(0,0,0,0.12),
         0 1px 4px rgba(0,0,0,0.12);border-radius:0px;position:relative;z-index:1100;width:100%;display:flex;min-height:64px;padding-left:24px;padding-right:24px;mui-prepared:;"
        >
          <button
            data-reactid=".1.0.1.0.0.0"
            style="border:10px;background:none;box-sizing:border-box;display:inline-block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);appearance:button;cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);position:relative;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;padding:12px;width:48px;height:48px;font-size:0;margin-top:8px;margin-right:8px;margin-left:-16px;mui-prepared:;"
            tabindex="0"
            type="button"
          >
            <div
              data-reactid=".1.0.1.0.0.0.0"
            >
              <svg
                data-reactid=".1.0.1.0.0.0.0.1:2:$/=10"
                style="display:inline-block;height:24px;width:24px;user-select:none;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;fill:#ffffff;color:#ffffff;mui-prepared:;"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                  data-reactid=".1.0.1.0.0.0.0.1:2:$/=10.0"
                />
              </svg>
            </div>
          </button>
          <div
            data-reactid=".1.0.1.0.0.1"
            style="white-space:nowrap;overflow:hidden;text-overflow:ellipsis;margin:0;padding-top:0;letter-spacing:0;font-size:24px;font-weight:400;color:#ffffff;line-height:64px;box-flex:1;flex:1;mui-prepared:;"
          >
            <span
              class="hidden-xs"
              data-reactid=".1.0.1.0.0.1.0"
            >
              <img
                alt="FirstVoices"
                data-reactid=".1.0.1.0.0.1.0.0"
                src="assets/images/logo.png"
                style="padding:0 0 5px 0;"
              />
            </span>
          </div>
          <div
            data-reactid=".1.0.1.0.0.3"
            style="float:left;position:relative;color:#fff;mui-prepared:;"
          >
            <div
              data-reactid=".1.0.1.0.0.3.$=10"
              style="display:inline-block;padding-right:10px;padding-top:15px;text-transform:uppercase;"
            >
              <a
                class="nav_link hidden-xs"
                data-reactid=".1.0.1.0.0.3.$=10.0"
                href="/nuxeo/app/explore/FV/sections/Data"
              >
                Choose a language
              </a>
            </div>
            <div
              data-reactid=".1.0.1.0.0.3.$=11"
              style="display:inline-block;padding:0 0 0 10px;"
            >
              <a
                class="nav_link"
                data-reactid=".1.0.1.0.0.3.$=11.0"
                href="https://firstvoices-dev.apps.prod.nuxeo.io/nuxeo/logout"
              >
                SIGN IN
              </a>
            </div>
            <span
              class="hidden"
              data-reactid=".1.0.1.0.0.3.$/=12"
              style="background-color:rgba(0, 0, 0, .175);display:inline-block;height:32px;margin-left:0;position:relative;top:12px;width:1px;float:none;color:rgba(0, 0, 0, .40);line-height:56px;margin-right:10px;mui-prepared:;"
            />
            <noscript
              data-reactid=".1.0.1.0.0.3.$=13"
            />
            <span
              class="search-bar-seperator"
              data-reactid=".1.0.1.0.0.3.$/=14"
              style="background-color:rgba(0, 0, 0, .175);display:inline-block;height:32px;margin-left:10px;position:relative;top:12px;width:1px;float:none;color:rgba(0, 0, 0, .40);line-height:56px;margin-right:0;mui-prepared:;"
            />
            <div
              class="hidden-xs"
              data-reactid=".1.0.1.0.0.3.$=15"
              style="background:#b40000;display:inline-block;"
            >
              <div
                class="fontAboriginalSans"
                data-reactid=".1.0.1.0.0.3.$=15.0"
                style="font-size:15px;line-height:1.5;width:inherit;height:48px;display:inline-block;position:relative;background-color:#fff;font-family:\\"Aboriginal Sans\\", \\"Aboriginal Serif\\", \\"Lucida Grande\\", \\"Lucida Sans Unicode\\", Gentium, Code2001;transition:height 200ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;margin-left:30px;padding-left:10px;padding-right:40px;mui-prepared:;"
              >
                <div
                  data-reactid=".1.0.1.0.0.3.$=15.0.1"
                  style="did-flip:true;position:absolute;opacity:1;color:rgba(0,0,0,0.3);transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;bottom:12px;"
                >
                  Search:
                </div>
                <input
                  data-reactid=".1.0.1.0.0.3.$=15.0.2"
                  id="mui-id-1"
                  name="searchbox"
                  style="tap-highlight-color:rgba(0,0,0,0);padding:0;position:relative;width:100%;height:100%;border:none;outline:none;background-color:transparent;color:#666666;font:inherit;mui-prepared:;"
                  type="text"
                />
                <div
                  data-reactid=".1.0.1.0.0.3.$=15.0.3"
                >
                  <hr
                    data-reactid=".1.0.1.0.0.3.$=15.0.3.0"
                    style="did-flip:true;border:none;border-bottom:solid 1px;border-color:#e0e0e0;bottom:8px;box-sizing:content-box;margin:0;position:absolute;width:79%;"
                  />
                  <hr
                    data-reactid=".1.0.1.0.0.3.$=15.0.3.1"
                    style="did-flip:true;border:none;border-bottom:solid 2px;border-color:#b40000;bottom:8px;box-sizing:content-box;margin:0;position:absolute;width:79%;transform:scaleX(0);transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;"
                  />
                </div>
              </div>
              <button
                class="hidden"
                data-reactid=".1.0.1.0.0.3.$=15.1"
                style="border:10px;background:none;box-sizing:border-box;display:inline-block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);appearance:button;cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);color:#ffffff;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;font-size:14px;letter-spacing:0;text-transform:uppercase;font-weight:500;border-radius:2px;user-select:none;position:relative;overflow:hidden;background-color:rgba(0, 0, 0, 0);line-height:36px;min-width:88px;padding:0;margin:0;mui-prepared:;"
                tabindex="0"
                type="button"
              >
                <div
                  data-reactid=".1.0.1.0.0.3.$=15.1.0"
                >
                  <span
                    data-reactid=".1.0.1.0.0.3.$=15.1.0.1"
                    style="position:relative;padding-left:16px;padding-right:16px;mui-prepared:;"
                  >
                    Cancel
                  </span>
                </div>
              </button>
            </div>
            <button
              data-reactid=".1.0.1.0.0.3.$=16"
              style="border:10px;background:none;box-sizing:border-box;display:inline-block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);appearance:button;cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);position:relative;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;padding:0px;width:48px;height:48px;font-size:0;top:7px;left:0;mui-prepared:;"
              tabindex="0"
              type="button"
            >
              <div
                data-reactid=".1.0.1.0.0.3.$=16.0"
              >
                <span
                  class="material-icons"
                  data-reactid=".1.0.1.0.0.3.$=16.0.1:1"
                  style="position:relative;font-size:24px;display:inline-block;user-select:none;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;color:#FFFFFF;fill:#666666;padding:3px;border-radius:20px;mui-prepared:;"
                >
                  search
                </span>
                <span
                  data-reactid=".1.0.1.0.0.3.$=16.0.1:2"
                >
                  search
                </span>
              </div>
            </button>
            <noscript
              data-reactid=".1.0.1.0.0.3.$=17"
            />
            <span
              class="locale-seperator"
              data-reactid=".1.0.1.0.0.3.$/=18"
              style="background-color:rgba(0, 0, 0, .175);display:inline-block;height:32px;margin-left:0;position:relative;top:12px;width:1px;float:none;color:rgba(0, 0, 0, .40);line-height:56px;margin-right:0;mui-prepared:;"
            />
            <button
              data-reactid=".1.0.1.0.0.3.$=19"
              style="border:10px;background:none;box-sizing:border-box;display:inline-block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);appearance:button;cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);position:relative;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;padding:0px;width:48px;height:48px;font-size:0;top:7px;left:0;mui-prepared:;"
              tabindex="0"
              type="button"
            >
              <div
                data-reactid=".1.0.1.0.0.3.$=19.0"
              >
                <span
                  class="material-icons"
                  data-reactid=".1.0.1.0.0.3.$=19.0.1:1"
                  style="position:relative;font-size:24px;display:inline-block;user-select:none;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;color:#FFFFFF;fill:#666666;padding:3px;border-radius:20px;mui-prepared:;"
                >
                  settings
                </span>
                <span
                  data-reactid=".1.0.1.0.0.3.$=19.0.1:2"
                >
                  settings
                </span>
              </div>
            </button>
          </div>
        </div>
        <div
          data-reactid=".1.0.1.0.1"
          style="box-sizing:border-box;-webkit-tap-highlight-color:rgba(0,0,0,0);background-color:rgb(171,0,0);height:56px;width:100%;padding:0px 24px;display:none;mui-prepared:;"
        >
          <div
            data-reactid=".1.0.1.0.1.0"
            style="float:right;position:relative;margin-left:-24px;mui-prepared:;"
          >
            <span
              data-reactid=".1.0.1.0.1.0.$/=10"
              style="padding-right:16px;line-height:56px;font-size:15px;display:inline-block;position:relative;float:left;color:#fff;padding:0 0 0 15px;mui-prepared:;"
            >
              Choose a language
            </span>
            <div
              data-reactid=".1.0.1.0.1.0.$/=11"
              style="display: inline-block; font-size: 15px; height: 48px; font-family: Arial, sans-serif; outline: none; position: relative; transition: all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms; float: left; color: rgba(0, 0, 0, 0.54); margin-right: 24px; width: auto;"
            >
              <div
                data-reactid=".1.0.1.0.1.0.$/=11.0"
                style="cursor:pointer;height:100%;position:relative;width:100%;mui-prepared:;"
              >
                <div
                  data-reactid=".1.0.1.0.1.0.$/=11.0.$=2=2before"
                  style="box-sizing:border-box;content:' ';display:table;mui-prepared:;"
                />
                <div
                  data-reactid=".1.0.1.0.1.0.$/=11.0.1:0"
                  style="did-flip:true;color:#fff;line-height:56px;opacity:1;position:relative;padding-left:24px;padding-right:48px;top:0;"
                />
                <svg
                  data-reactid=".1.0.1.0.1.0.$/=11.0.1:1"
                  style="display:inline-block;height:24px;width:24px;user-select:none;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;fill:#e0e0e0;position:absolute;right:16px;top:16px;mui-prepared:;"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                    data-reactid=".1.0.1.0.1.0.$/=11.0.1:1.0"
                  />
                </svg>
                <div
                  data-reactid=".1.0.1.0.1.0.$/=11.0.1:2"
                  style="did-flip:true;border-top:solid 1px #e0e0e0;bottom:1px;left:0;margin:-1px 24px;right:0;position:absolute;"
                />
                <div
                  data-reactid=".1.0.1.0.1.0.$/=11.0.$=2=2after"
                  style="box-sizing:border-box;content:' ';clear:both;display:table;mui-prepared:;"
                />
              </div>
              <noscript
                data-reactid=".1.0.1.0.1.0.$/=11.1"
              />
            </div>
          </div>
        </div>
        <div
          data-reactid=".1.0.1.0.2"
        >
          <div
            data-reactid=".1.0.1.0.2.1"
            style="background-color:#ffffff;transition:transform 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;box-sizing:border-box;font-family:Arial, sans-serif;-webkit-tap-highlight-color:rgba(0,0,0,0);box-shadow:0 3px 10px rgba(0,0,0,0.16),
         0 3px 10px rgba(0,0,0,0.23);border-radius:0px;height:auto;width:256px;position:fixed;z-index:1300;left:0;top:0;transform:translate3d(-266px, 0, 0);overflow:auto;mui-prepared:;"
          >
            <div
              data-reactid=".1.0.1.0.2.1.1:0"
              style="background-color:#b40000;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;box-sizing:border-box;font-family:Arial, sans-serif;-webkit-tap-highlight-color:rgba(0,0,0,0);box-shadow:0 1px 6px rgba(0,0,0,0.12),
         0 1px 4px rgba(0,0,0,0.12);border-radius:0px;position:relative;z-index:1100;width:100%;display:flex;min-height:64px;padding-left:24px;padding-right:24px;mui-prepared:;"
            >
              <div
                data-reactid=".1.0.1.0.2.1.1:0.0"
                style="margin-top:8px;margin-right:8px;margin-left:-16px;mui-prepared:;"
              >
                <button
                  data-reactid=".1.0.1.0.2.1.1:0.0.0"
                  style="border:10px;background:none;box-sizing:border-box;display:inline-block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);appearance:button;cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);position:relative;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;padding:12px;width:48px;height:48px;font-size:0;mui-prepared:;"
                  tabindex="0"
                  type="button"
                >
                  <div
                    data-reactid=".1.0.1.0.2.1.1:0.0.0.0"
                  >
                    <svg
                      data-reactid=".1.0.1.0.2.1.1:0.0.0.0.1:2:$/=10"
                      style="display:inline-block;height:24px;width:24px;user-select:none;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;fill:#ffffff;color:#ffffff;mui-prepared:;"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                        data-reactid=".1.0.1.0.2.1.1:0.0.0.0.1:2:$/=10.0"
                      />
                    </svg>
                  </div>
                </button>
              </div>
              <div
                data-reactid=".1.0.1.0.2.1.1:0.1"
                style="white-space:nowrap;overflow:hidden;text-overflow:ellipsis;margin:0;padding-top:0;letter-spacing:0;font-size:24px;font-weight:400;color:#ffffff;line-height:64px;box-flex:1;flex:1;mui-prepared:;"
              >
                <img
                  alt="FirstVoices"
                  data-reactid=".1.0.1.0.2.1.1:0.1.0"
                  src="assets/images/logo.png"
                  style="padding:0 0 5px 0;"
                />
              </div>
            </div>
            <div
              data-reactid=".1.0.1.0.2.1.1:1"
              style="background-color:#ffffff;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;box-sizing:border-box;font-family:Arial, sans-serif;-webkit-tap-highlight-color:rgba(0,0,0,0);border-radius:2px;padding:0;padding-bottom:8px;padding-top:8px;mui-prepared:;"
            >
              <div
                data-reactid=".1.0.1.0.2.1.1:1.1:$1/=10=2$home"
              >
                <span
                  class="1"
                  data-reactid=".1.0.1.0.2.1.1:1.1:$1/=10=2$home.0"
                  style="border:10px;background:none;box-sizing:border-box;display:block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);color:#666666;font-size:16px;line-height:16px;position:relative;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;mui-prepared:;"
                  tabindex="0"
                  type="button"
                  value="/nuxeo/app/"
                >
                  <div
                    data-reactid=".1.0.1.0.2.1.1:1.1:$1/=10=2$home.0.0"
                  >
                    <div
                      data-reactid=".1.0.1.0.2.1.1:1.1:$1/=10=2$home.0.0.1"
                      style="margin-left:0;padding-left:16px;padding-right:16px;padding-bottom:16px;padding-top:16px;position:relative;mui-prepared:;"
                    >
                      <div
                        data-reactid=".1.0.1.0.2.1.1:1.1:$1/=10=2$home.0.0.1.$primaryText"
                        style="mui-prepared:;"
                      >
                        Home
                      </div>
                    </div>
                  </div>
                </span>
              </div>
              <div
                data-reactid=".1.0.1.0.2.1.1:1.1:$2/=10=2$get-started"
              >
                <span
                  class="1"
                  data-reactid=".1.0.1.0.2.1.1:1.1:$2/=10=2$get-started.0"
                  style="border:10px;background:none;box-sizing:border-box;display:block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);color:#666666;font-size:16px;line-height:16px;position:relative;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;mui-prepared:;"
                  tabindex="0"
                  type="button"
                  value="/nuxeo/app/content/get-started"
                >
                  <div
                    data-reactid=".1.0.1.0.2.1.1:1.1:$2/=10=2$get-started.0.0"
                  >
                    <div
                      data-reactid=".1.0.1.0.2.1.1:1.1:$2/=10=2$get-started.0.0.1"
                      style="margin-left:0;padding-left:16px;padding-right:16px;padding-bottom:16px;padding-top:16px;position:relative;mui-prepared:;"
                    >
                      <div
                        data-reactid=".1.0.1.0.2.1.1:1.1:$2/=10=2$get-started.0.0.1.$primaryText"
                        style="mui-prepared:;"
                      >
                        Get started
                      </div>
                    </div>
                  </div>
                </span>
              </div>
              <div
                data-reactid=".1.0.1.0.2.1.1:1.1:$3/=10=2$explore"
              >
                <span
                  class="1"
                  data-reactid=".1.0.1.0.2.1.1:1.1:$3/=10=2$explore.0"
                  style="border:10px;background:none;box-sizing:border-box;display:block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);color:#666666;font-size:16px;line-height:16px;position:relative;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;mui-prepared:;"
                  tabindex="0"
                  type="button"
                  value="/nuxeo/app/explore/FV/sections/Data"
                >
                  <div
                    data-reactid=".1.0.1.0.2.1.1:1.1:$3/=10=2$explore.0.0"
                  >
                    <div
                      data-reactid=".1.0.1.0.2.1.1:1.1:$3/=10=2$explore.0.0.1"
                      style="margin-left:0;padding-left:16px;padding-right:16px;padding-bottom:16px;padding-top:16px;position:relative;mui-prepared:;"
                    >
                      <div
                        data-reactid=".1.0.1.0.2.1.1:1.1:$3/=10=2$explore.0.0.1.$primaryText"
                        style="mui-prepared:;"
                      >
                        Explore languages
                      </div>
                    </div>
                  </div>
                </span>
              </div>
              <div
                data-reactid=".1.0.1.0.2.1.1:1.1:$4/=10=2$kids"
              >
                <span
                  class="1"
                  data-reactid=".1.0.1.0.2.1.1:1.1:$4/=10=2$kids.0"
                  style="border:10px;background:none;box-sizing:border-box;display:block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);color:#666666;font-size:16px;line-height:16px;position:relative;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;mui-prepared:;"
                  tabindex="0"
                  type="button"
                  value="/nuxeo/app/kids"
                >
                  <div
                    data-reactid=".1.0.1.0.2.1.1:1.1:$4/=10=2$kids.0.0"
                  >
                    <div
                      data-reactid=".1.0.1.0.2.1.1:1.1:$4/=10=2$kids.0.0.1"
                      style="margin-left:0;padding-left:16px;padding-right:16px;padding-bottom:16px;padding-top:16px;position:relative;mui-prepared:;"
                    >
                      <div
                        data-reactid=".1.0.1.0.2.1.1:1.1:$4/=10=2$kids.0.0.1.$primaryText"
                        style="mui-prepared:;"
                      >
                        Kids
                      </div>
                    </div>
                  </div>
                </span>
              </div>
              <div
                data-reactid=".1.0.1.0.2.1.1:1.1:$5/=10=2$contribute"
              >
                <span
                  class="1"
                  data-reactid=".1.0.1.0.2.1.1:1.1:$5/=10=2$contribute.0"
                  style="border:10px;background:none;box-sizing:border-box;display:block;font:inherit;font-family:Arial, sans-serif;tap-highlight-color:rgba(0, 0, 0, 0);cursor:pointer;text-decoration:none;outline:none;transform:translate3d(0, 0, 0);color:#666666;font-size:16px;line-height:16px;position:relative;transition:all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms;mui-prepared:;"
                  tabindex="0"
                  type="button"
                  value="/nuxeo/app/content/contribute"
                >
                  <div
                    data-reactid=".1.0.1.0.2.1.1:1.1:$5/=10=2$contribute.0.0"
                  >
                    <div
                      data-reactid=".1.0.1.0.2.1.1:1.1:$5/=10=2$contribute.0.0.1"
                      style="margin-left:0;padding-left:16px;padding-right:16px;padding-bottom:16px;padding-top:16px;position:relative;mui-prepared:;"
                    >
                      <div
                        data-reactid=".1.0.1.0.2.1.1:1.1:$5/=10=2$contribute.0.0.1.$primaryText"
                        style="mui-prepared:;"
                      >
                        Contribute
                      </div>
                    </div>
                  </div>
                </span>
              </div>
            </div>
            <hr
              data-reactid=".1.0.1.0.2.1.1:2"
              style="did-flip:true;margin:0;margin-top:-1px;margin-left:0;height:1px;border:none;background-color:#e0e0e0;"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      data-reactid=".1.0.2"
      id="pageContainer"
    >
      <div
        data-reactid=".1.0.2.0"
      >
        <div
          class="breadcrumbContainer row"
          data-reactid=".1.0.2.0.0"
        >
          <div
            class="clearfix"
            data-reactid=".1.0.2.0.0.0"
            style="background-color:#e1e1e2;"
          />
        </div>
        <div
          class="page-default-theme"
          data-reactid=".1.0.2.0.1"
        >
          <div
            data-reactid=".1.0.2.0.1.0"
          >
            <div
              class="row"
              data-reactid=".1.0.2.0.1.0.0"
            >
              <div
                class="col-xs-12"
                data-reactid=".1.0.2.0.1.0.0.0"
              >
                <h1
                  data-reactid=".1.0.2.0.1.0.0.0.0"
                >
                  404 - Page not found
                </h1>
              </div>
            </div>
            <div
              class="row"
              data-reactid=".1.0.2.0.1.0.1"
            >
              <div
                class="col-xs-12"
                data-reactid=".1.0.2.0.1.0.1.0"
              >
                <div
                  data-reactid=".1.0.2.0.1.0.1.0.0"
                >
                  <p
                    data-reactid=".1.0.2.0.1.0.1.0.0.0"
                  >
                    <span
                      data-reactid=".1.0.2.0.1.0.1.0.0.0.0"
                    >
                      Please report this error so that we can fix it
                    </span>
                    <span
                      data-reactid=".1.0.2.0.1.0.1.0.0.0.1"
                    >
                      .
                    </span>
                  </p>
                  <p
                    data-reactid=".1.0.2.0.1.0.1.0.0.1"
                  >
                    <span
                      data-reactid=".1.0.2.0.1.0.1.0.0.1.0"
                    >
                      Include what link or action you took to get to this page
                    </span>
                    <span
                      data-reactid=".1.0.2.0.1.0.1.0.0.1.1"
                    >
                      .
                    </span>
                  </p>
                  <p
                    data-reactid=".1.0.2.0.1.0.1.0.0.2"
                  >
                    Thank You!
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="row"
      data-reactid=".1.0.3"
      id="pageFooter"
    >
      <div
        data-reactid=".1.0.3.0"
      >
        <footer
          class="footer footer-default-theme"
          data-reactid=".1.0.3.0.0"
        >
          <div
            class="container-fluid"
            data-reactid=".1.0.3.0.0.0"
          >
            <div
              class="row"
              data-reactid=".1.0.3.0.0.0.0"
            >
              <div
                class="col-xs-12 col-md-5 col-md-offset-1"
                data-reactid=".1.0.3.0.0.0.0.0"
                style="padding-top:20px;"
              >
                <img
                  alt="FirstVoices Logo"
                  class="pull-left"
                  data-reactid=".1.0.3.0.0.0.0.0.0"
                  src="assets/images/logo-fpcc-white.png"
                />
              </div>
              <div
                class="col-xs-12 col-md-5 col-md-offset-1 body"
                data-reactid=".1.0.3.0.0.0.0.1"
                style="padding-top:20px;font-weight:100;"
              >
                <p
                  data-reactid=".1.0.3.0.0.0.0.1.0"
                >
                  <a
                    data-reactid=".1.0.3.0.0.0.0.1.0.0"
                    href="/nuxeo/app/content/disclaimer/"
                  >
                    Disclaimer
                  </a>
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.0.1"
                  >
                     | 
                  </span>
                  <a
                    data-reactid=".1.0.3.0.0.0.0.1.0.2"
                    href="/nuxeo/app/content/conditions/"
                  >
                    Conditions of use
                  </a>
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.0.3"
                  >
                     | 
                  </span>
                  <a
                    data-reactid=".1.0.3.0.0.0.0.1.0.4"
                    href="https://firstvoices.atlassian.net/servicedesk/customer/portal/1/create/16"
                  >
                    Feedback
                  </a>
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.0.5"
                  >
                     | 
                  </span>
                  <a
                    data-reactid=".1.0.3.0.0.0.0.1.0.6"
                    href="http://fpcf.ca/donate-now/"
                  >
                    Donate
                  </a>
                </p>
                <p
                  data-reactid=".1.0.3.0.0.0.0.1.1"
                >
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.1.0"
                  >
                    Phone
                  </span>
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.1.1"
                  >
                     : +1-250-652-5952 · 
                  </span>
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.1.2"
                  >
                    Email
                  </span>
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.1.3"
                  >
                     : support@fpcc.ca
                  </span>
                </p>
                <p
                  data-reactid=".1.0.3.0.0.0.0.1.2"
                >
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.2.0"
                  >
                    © 2000-
                  </span>
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.2.1"
                  >
                    2019
                  </span>
                  <span
                    data-reactid=".1.0.3.0.0.0.0.1.2.2"
                  >
                     FirstVoices
                  </span>
                </p>
              </div>
            </div>
          </div>
        </footer>
        <div
          class="container-fluid"
          data-reactid=".1.0.3.0.1"
          style="background-color:#0d6c80;border-top:1px solid #1c788c;"
        >
          <div
            class="row"
            data-reactid=".1.0.3.0.1.0"
          >
            <div
              class="col-xs-12"
              data-reactid=".1.0.3.0.1.0.0"
            >
              <p
                data-reactid=".1.0.3.0.1.0.0.0"
                style="font-size:10px;line-height:130%;margin-top:10px;color:#4191a5;"
              >
                <span
                  data-reactid=".1.0.3.0.1.0.0.0.0"
                >
                  © 
                </span>
                <span
                  data-reactid=".1.0.3.0.1.0.0.0.1"
                >
                  This database is protected by copyright laws and is owned by the First Peoples’ Cultural Foundation. All materials on this site are protected by copyright laws and are owned by the individual Indigenous language communities who created the archival content. Language and multimedia data available on this site is intended for private, non-commercial use by individuals. Any commercial use of the language data or multimedia data in whole or in part, directly or indirectly, is specifically forbidden except with the prior written authority of the owner of the copyright.
                </span>
                <span
                  data-reactid=".1.0.3.0.1.0.0.0.2"
                >
                  .
                </span>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
